changelogs/unreleased/1888-remove-depracated-OptionVariable.yml

```
@@ -0,0 +1,7 @@
+---
+description: Remove support for leaving nullable attribute unassigned, an exception will now be raised. You should make sure optional variables are always assigned a value.
+change-type: major
+issue-nr: 1888
+destination-branches: [master, iso5]
```

```suggestion
destination-branches: [master]
```
The merge tool wouldn't even allow this: major changes can only go into master